pom.xml

```diff
@@ -1,272 +1,345 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="http://maven.apache.org/POM/4.0.0"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Any particular reason you changed the indentation from 2 to 4 spaces in this file? Makes it a bit more difficult to see what has changed.

Reply:
Although I almost forgot about github's ignore whitespace toggle :)

Reply:
Yeah I just auto-formatted this.